lib/livebook/evaluator.ex

```diff
@@ -68,6 +68,30 @@ defmodule Livebook.Evaluator do
     end
   end
 
+  @doc """
+  Computes the memory usage from this evaluator node.
+  """
+  def memory do
```

Comment:
```suggestion
  @spec memory() :: Livebook.Runtime.runtime_memory()
  def memory do
```

Since we are ignoring `:system` we should remove it from `Runtime.runtime_memory`, also I'd rename it to `Runtime.memory_usage`.

Reply:
I will keep it `runtime_memory()` because we may need to measure `system_memory` too if we allow running on separate machines.

Reply:
In fact, if we connect to a remote node today, we will show the wrong information... 🤔